test-data/run.test

```diff
@@ -491,11 +496,24 @@ def h() -> None:
 
 def i() -> None:
     try:
-        r(True)
+        r(0)
     except:
-        print(str(wrapsys.exc_info()[1]))
+        print(type(wrapsys.exc_info()[1]))
         raise
 
+def j(n: int) -> None:
```

Comment:
What about a try/except inside an except block of another try statement? If this doesn't work yet, you can file a separate issue about this.